tests/ui/async-await/await-keyword/incorrect-syntax-suggestions.stderr

```diff
@@ -2,115 +2,214 @@ error: incorrect use of `await`
   --> $DIR/incorrect-syntax-suggestions.rs:8:13
    |
 LL |     let _ = await bar();
-   |             ^^^^^^^^^^^ help: `await` is a postfix operation: `bar().await`
+   |             ^^^^^^^^^^^
+   |
+help: `await` is a postfix operation
+   |
+LL |     let _ = bar().await;
+   |             ~~~~~~~~~~~
```

Comment:
I guess it would make sense to make this a verbose suggestion if the `await ` removal were one span and the `.await` addition another.

But the way it is currently it just seems redundant.